bika/lims/api.py

```diff
@@ -1052,6 +1052,37 @@ def get_current_user():
     return ploneapi.user.get_current()
 
 
+def get_user_contact(user, contact_types=None):
+    """Returns the current contact associated to the user passed in, if any.
+    If the user passed in has no contact associated, return None. If
+    contact_types is not specified, searches for 'Contact' and 'LabContact'
+    :param: Plone user
+    :contact_types: array with the contact types to look for
+    :returns: Contact associated to the Plone user, if any. Otherwise, None
+    """
```

Comment:
Make the first line of your docstring a quick-info.
Then add a blank line and add a more elaborative description.
Add another blank like and add the parameters.
Also see: https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

So I would like to have it like this:

```python
def get_user_contact(user, contact_types=['Contact', 'LabContact']):
    """Returns the associated contact of a Plone user

    If the user passed in has no contact associated, return None.
    The `contact_types` parameter filter the portal types for the search.
 
    :param: Plone user
    :contact_types: List with the contact portal types to search
    :returns: Contact associated to the Plone user or None
    """
```